tests/init.go

```diff
@@ -0,0 +1,60 @@
+package tests
+
+import (
+	"flag"
+	"regexp"
+	"strings"
+
+	"github.com/ipfs/gateway-conformance/tooling/specs"
+)
+
+type specsFlag string
+
+func (s *specsFlag) String() string {
+	return string(*s)
+}
+
+func (s *specsFlag) Set(value string) error {
+	names := strings.Split(value, ",")
+	var only, enable, disable = []specs.Spec{}, []specs.Spec{}, []specs.Spec{}
+	for _, name := range names {
+		spec, err := specs.FromString(regexp.MustCompile(`^[-+]`).ReplaceAllString(name, ""))
+		if err != nil {
+			return err
+		}
+		if strings.HasPrefix(name, "+") {
```

Comment:
If a spec is prefixed with `+`, it should be enabled. This can be used to enable tests for specs that are disabled by default (those not mature enough).

For example, let's say spec `fancy` has maturity `wip`. If we run tests without any flags, then tests for `fancy` would be skipped. If we used `-specs=+fancy`, tests for `fancy` would be executed AND all the tests that are enabled by default would be executed too.

Reply:
Should we keep these comments somewhere in the README?